src/commands/synthetics/reporters/default.ts

```diff
@@ -255,9 +257,12 @@ const getResultIconAndColor = (resultOutcome: ResultOutcome): [string, chalk.Cha
 }
 
 export class DefaultReporter implements MainReporter {
+  private testWaitSpinner?: ora.Ora
+  private writable: Writable
   private write: Writable['write']
 
   constructor({context}: {context: BaseContext}) {
+    this.writable = context.stdout
```

Comment:
Nit? Should we replace `this.write(...)` with `this.writable.write(...)` to avoid the potential confusion between `write` and `writable` in this module, and simplify slightly by removing a property? 🤔

Reply:
Since `write` is widely used I think it's nice to have it as a property directly. But agreed the two of them is a bit confusing, we could store the context instead